protocols/illumina-nextera-XT-library-prep-part1/README.md

```diff
@@ -0,0 +1,54 @@
+# Nextera XT DNA Library Prep Kit Protocol: Part 1/4 - Tagment Genomic DNA and Amplify Libraries
+
+### Author
+[Opentrons](http://www.opentrons.com/)
+
+### Partner
+[CosmosID](http://www.cosmosid.com/)
+
+## Categories
```

Comment:
https://raw.githubusercontent.com/Opentrons/Protocols/protocol-delivery/Template/README.md is our working reference of how READMEs should be formatted

Categories must look like:
```md
## Categories
* Sample Prep
	* Sequencing
```

Where to the left of Sequencing is a tab, not spaces

If this is wrong, the protocol might not show up in search results, and/or the categories listing on the site might disappear or look wrong

Reply:
Changes submitted! Updated categories.

Reply:
New category should appear as:
```
## Categories
* NGS Library Prep
	* Illumina
```